Content.Shared/_RMC14/Xenonids/Egg/XenoEggSystem.cs

```diff
@@ -229,13 +230,49 @@ private void OnXenoEggAfterInteract(Entity<XenoEggComponent> egg, ref AfterInter
     private void OnXenoEggActivateInWorld(Entity<XenoEggComponent> egg, ref ActivateInWorldEvent args)
     {
         // TODO RMC14 multiple hive support
-        if (!HasComp<XenoComponent>(args.User) || !HasComp<HandsComponent>(args.User))
+        if (!HasComp<XenoParasiteComponent>(args.User) && (!HasComp<XenoComponent>(args.User) || !HasComp<HandsComponent>(args.User)))
```

Comment:
Is this change necessary or a leftover from something else?

Reply:
Parasites don't have hands so I added it to skip over the early return.